app/mailers/notifications_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class NotificationsMailer < BaseMailer
+  default from: NOREPLY_EMAIL
+
+  def pending_cards(user)
+    @user = user
+
+    mail(to: @user.email, subject: 'You have the cards for check')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.